types/tests/server.test.ts

```diff
@@ -145,3 +154,17 @@ createOrder({
         },
     ],
 });
+
+updateOrder({
```

Comment:
you might think about adding a case for `move` and `remove` as well

Reply:
Test added for "remove". I ended up removing the type for "move" since I don't see any attributes supporting that operation.